server/app/routes/checkout/index.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+var mongoose = require('mongoose');
+var router = require('express').Router();
+module.exports = router;
+
+var cartModel = mongoose.model('Cart');
+var lineItemModel = mongoose.model('LineItem');
+var orderModel = mongoose.model('Order');
+
+router.post('/', function (req, res, next) {
+	var cartId = req.body.cartInfo._id;
+	var sessionId = req.sessionID;
+
+	// Function to generate a random confirmation number
+
+	// It should create an order that includes the line items
+
+	console.log("The cartId is ", cartId);
+
+	cartModel.findOne({_id: cartId}).exec(function (err, currentCart) {
+		if (err) throw err;
+
+		console.log("The currentCart is ", currentCart);
+
+		var order = new orderModel({
+			sessionId: sessionId,
+			status: "Active",
+			user: "5541421a6c6c31d47876e032",
```

Comment:
Again, some of these fields are hard-coded for now, but allows us to simulate the experience.